### DISCUSSION TITLE
.yml error

### DISCUSSION THREAD

**Discussion by mgsnell** [blakeblackshear/frigate#1971] [Ask A Question] [answered]
I am trying to get Frigate up and running but keep running into a .yml error  

"Error parsing config: 3 validation errors for FrigateConfig
cameras -> porch_1 -> fps
  extra fields not permitted (type=value_error.extra)
cameras -> porch_1 -> height
  extra fields not permitted (type=value_error.extra)
cameras -> porch_1 -> width
  extra fields not permitted (type=value_error.extra)"

I have run this thru an on-line yaml checker and it is good

                   
[frigate.yml.pdf](https://github.com/blakeblackshear/frigate/files/7316093/frigate.yml.pdf)

**Comment by blakeblackshear** [comment, marked as answer]
It's valid yaml, but it's not a valid configuration file based on the documentation. See here: https://docs.frigate.video/configuration/index

**Reply by mgsnell** [reply]
Thanks, was following an online tutorial did (or thought I did) everything they showed and got this. Did what the instructions you referred me to and now it works.

**Reply by blakeblackshear** [reply]
This release is very new, so all the video tutorials are unfortunately outdated now.
